### ISSUE TITLE
lower performance than the paper results

### ISSUE THREAD

**Issue by caddy-liu** [ayyyq/TARA#8]
Hi.
Thanks for your code! According to the paper, results using AMRBART on WikiEvents test set based on RoBERTa large is as bellow.
`78.35	76.29	73.07	70.83
`

But I only received low performance as follow:
```
'========for Head F1 Identification========' 
Precision: 71.9361 Recall: 81.4889 F1: 76.4151
'========for COREF F1 Identification========' 
Precision: 70.6927 Recall: 80.0805 F1: 75.0943
'========for Head F1 Classification========' 
Precision: 67.1403 Recall: 76.0563 F1: 71.3208
'========for COREF F1 Classification========' 
Precision: 66.0746 Recall: 74.8491 F1: 70.1887
```
It puzzled me. I set the same hyperparameters according to the appendix and use the preprocessed data provided by you . Did I do something wrong? 

Thanks for reply.

**Comment by ayyyq** [comment]
Hi, sorry for the late response. Has your issue been resolved? If you are still concerned about this issue, were you able to successfully reproduce the results of transition-AMR on WikiEvents, or AMRBART on RAMS? Your response will help us identify the problem. Thank you!

**Comment by caddy-liu** [comment]
Thank you for reply.
The issue hasn't been resolved.
I use the preprocessed data provided by you. `amrbart-dglgraph-train.pkl` etc.
And I reproduce the results of AMRBART on WikiEvents by using the resource code.

**Comment by ayyyq** [comment]
Could you please let me know if you were able to successfully reproduce the results of transition-AMR on WikiEvents, or AMRBART on RAMS? This will help me determine whether the issue is due to the preprocessed data provided, the training code, or perhaps some other reason.

**Comment by caddy-liu** [comment]
OK.
All results reproduced by me are lower than the paper results.
The results of transition-AMR on WikiEvents are almost `76.5840 75.1148 70.7071 69.6051,` and the results of  AMRBART on RAMS are almost `56.2098 51.5586 58.4364`.
Thanks for reply.

**Comment by ayyyq** [comment]
Thank you for your results.

~First, we apologize as we found that the code `src_wikievents` is actually the `wo surrounding events` setting in ablation study. We will upload the complete `src_wikievents` code as soon as possible (this does not affect the experimental results on RAMS).~

Second, our previous experiments with AMRBART on RAMS (which were not reported in the original paper) yielded the following results for Span F1 (Dev/Test) and Head F1 (Dev/Test): 50.10/51.37, 57.81/59.32. Therefore, within an acceptable margin of error, the results for transition-AMR on WikiEvents with `wo surrounding events` and AMRBART on RAMS are reasonable.

However, we must admit that your results of AMRBART on WikiEvents are much lower than those we have achieved. We will verify the uploaded preprocessed data again, but since the data on the server has been cleared, this process may take a bit longer. You can generate AMRBART for WikiEvents from scatch by following the instructions as https://github.com/ayyyq/TARA/tree/main/data_processing.

**Comment by caddy-liu** [comment]
Thank you very much for your reply. My troubles have been resolved.

**Comment by ayyyq** [comment]
Hi, after reviewing the code once more, I can confirm that the uploaded code and data are complete and without issues (my apologies for the earlier misleading information about `src_wikievents`).

You can access our full conda environment (`fastnlp.yml`) and the outputs (`checkpoints/`) for two settings on wikievents: transition-AMR and AMRBART, through [this link](https://drive.google.com/drive/folders/1R6mIFFtGcF_d-Pjkeu-xPnOWXcp0zKHW?usp=sharing). For each setting, the two checkpoints with the highest dev scores have been saved, and the test results of the checkpoint with the higest dev score have been recorded in `evaluate_results.log`. Specifically, the test results for `wikievents_roberta-large_transition` are `78.01, 76.05, 72.66, 71.24`; and for `wikievents_roberta-large_amrbart`, the results are `77.56, 74.98, 71.47, 69.80`.

Please note that variations in the **experimental environment** (such as GPUs and package versions) may affect the results. Additionally, since we use the **checkpoints with the highest dev scores** to calculate the test scores on wikievents, it may lead to more significant fluctuations in the test results. I recommend conducting repeated experiments or adjusting `monitor = f#f#test` in [this line of the code](https://github.com/ayyyq/TARA/blob/59621f6bfec9184d4a68572bfc74f7fd03d2984b/src_wikievents/train.py#L255). Thank you for your attention.
